### PR TITLE
Lerch updates

### PR DESCRIPTION
**Pull request by alexanderlerch**
Merge all recent dataset updates and additions

### CODE REVIEW

**Comment by lostanlen** [comment]
Many thanks @alexanderlerch for getting to this. Do you know why Travis CI is failing? Can I help?

**Comment by alexanderlerch** [comment]
@lostanlen Sorry, I have never looked at let alone ran the test script.

File: mir-datasets.yaml [hunk 1]
```diff
@@ -200,6 +222,23 @@ Cross-Era:
...
 
+CSD:
+  url: https://zenodo.org/record/2649950
+  title: Choral Singing Dataset
+  metadata:
+    - f0
+    - MIDI
+  contents: 48 recordings
+  audio: 'yes'
+<a title="Choral Singing Dataset" href="" target="_blank" rel="noopener">CSD</a> | pitch | 48 recordings | yes
```
Comment (stefan-balke) [inline mir-datasets.yaml:233]: please remove this

File: mir-datasets.yaml [hunk 2]
```diff
@@ -225,6 +270,12 @@ DEAPDataset:
   contents: 120 music video excerpts
   audio: 'no'
 
+DESED:
+  url: https://github.com/turpaultn/DESED
+  metadata:10 audio event classes
```
Comment (stefan-balke) [inline mir-datasets.yaml:275]: please add space

**Comment by alexanderlerch** [comment]
Hmm. What's the error now?

**Comment by stefan-balke** [comment]
Check out in the travis build, there you have the error message per URL.
Some 403 and 500s.

Do we have a decision about these?

**Comment by stefan-balke** [comment]
I guess we put those to the `archive.yaml` which are no longer available

**Comment by stefan-balke** [comment]
Let me check #22 first.

**Comment by stefan-balke** [comment]
#28 is merged. Could you please rebase?

**Comment by stefan-balke** [comment]
No worries, merged the new master changes...let's see.

**Comment by stefan-balke** [comment]
Alright, merging. Please don't call the Git-Police :)
Thanks @alexanderlerch

**Pull request merged**